grafana-tikv-dashboard.md

```diff
@@ -100,7 +100,14 @@ This document provides a detailed description of these key metrics on the **TiKV
 
 ## Raft process
 
-- Ready handled: The count of handled ready operations per second
+- Ready handled: The number of handled ready operations per type per second
+    - count: The number of handled ready operations
+    - has_ready_region: The number of Regions who have ready
+    - pending_region: The operations of the Region being checked whether has ready. This metric is deprecated since v3.0.0
```

Comment:
```suggestion
    - pending_region: The operations per second of the Regions being checked for whether it has ready. This metric is deprecated since v3.0.0
```